crowdin_updater.py

```diff
@@ -57,18 +57,21 @@ def download(self, outputdir, build=True, package=None):
             zipdata = StringIO(r.content)
             with zipfile.ZipFile(zipdata) as zf:
                 for filepath in zf.namelist():
-                    language = os.path.dirname(filepath)
-                    filename = os.path.basename(filepath)
-                    if filename:
-                        if self.format == 'json':
-                            root, ext = os.path.splitext(filename)
-                            target_filename = '%s-%s%s' % (root, language, ext)
-                        else:
-                            target_filename = filename
-                        target = file(os.path.join(outputdir, target_filename), "wb")
-                        source = zf.open(filepath)
-                        with source, target:
-                            shutil.copyfileobj(source, target)
+                    # let's skip folder entries
+                    if not filepath.endswith('/'):
```

Comment:
ok, i forgot that point, please just explicit in a comment the structure that you're parsing, then